src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -266,7 +275,11 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, EntityFilter
   color: $BCgovGold9;
 }
 
-.v-icon.mdi-information-outline {
-  margin-top: -2px;
+.v-icon {
+  &.mdi-information-outline,
+  &.mdi-account,
+  &.mdi-domain {
+    margin-top: -2px;
+  }
```

Comment:
Is this specific to these icons so it excludes others within this component?

Reply:
Exactly. The other icons are fine.